controllers/octaviaapi_controller.go

```diff
@@ -811,7 +812,8 @@ func (r *OctaviaAPIReconciler) reconcileNormal(ctx context.Context, instance *oc
 		return ctrl.Result{}, err
 	}
 	instance.Status.ReadyCount = depl.GetDeployment().Status.ReadyReplicas
-	if instance.Status.ReadyCount > 0 {
+	if instance.Status.ReadyCount > 0 &&
```

Comment:
we need to compare to the instance.Spec.replicas instead of > 0 otherwise we can mark the update ready before every pods are running with the new image.

Reply:
This makes sense to me, I'll change to be consistent w/ the other operators.